lib/views/profile_screen.dart

```diff
@@ -84,9 +84,10 @@ class ProfilePageState extends ConsumerState<ProfilePage> {
                     leading: CircleAvatar(
                       radius: 30,
                       backgroundImage: NetworkImage(
-                        profileData.profilePictureUrl ?? ""),
+                        profileData.profilePictureUrl ?? "https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_1280.png"),
```

Comment:
The default image for `profilePictureUrl` is already set on the User class, I don't think the null check is necessary?

Reply:
It's just in case something goes wrong with the value, I'd rather have it be the default profile picture than it being a blank url